### ISSUE TITLE
SIGPIPE caught! found in goaccess 

### ISSUE THREAD

**Issue by narayan1599** [allinurl/goaccess#1755]
it is showing at every end month in my screen terminal SIGPIPE caught! then it the page static 
and stop realtime data

**Comment by allinurl** [comment]
It's okay to get a SIGPIPE as goaccess should take care of it. Can you post the output of running it via `ltrace` , e.g.,  

    ltrace -p <goaccess_pid> -o debug.log

and/or `strace` Thanks.

**Comment by narayan1599** [comment]
![Screenshot 2020-05-05 at 11 37 58 PM](https://user-images.githubusercontent.com/43582408/81100041-7b1ab380-8f29-11ea-8e84-e73a3d09b064.png)

do u want this output

**Comment by allinurl** [comment]
Yes, however, please let goaccess run for sometime while logging to ltrace and then attach the output here. Thanks

**Comment by narayan1599** [comment]
> It's okay to get a SIGPIPE as goaccess should take care of it. Can you post the output of running it via `ltrace` , e.g.,
> 
> ```
> ltrace -p <goaccess_pid> -o debug.log
> ```
> 
> and/or `strace` Thanks.

should I run this command also goaccess command?

**Comment by allinurl** [comment]
Yes, both of them. Either you could run ltrace from the beginning or simply attach goaccess pid once it has finished parsing the initial log.
